spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+describe PostsController, type: :request do
+  describe '#index' do
+    it 'returns a successful response' do
+      get '/users/1/posts'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get '/users/1/posts'
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get '/users/1/posts'
+      expect(response.body).to include('Tom')
+    end
+  end
+
+  describe '#show' do
+    it 'returns a successful response' do
+      get '/users/1/posts/1'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      get '/users/1/posts/1'
+      expect(response).to render_template(:show)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get '/users/1/posts/1'
+      expect(response.body).to include('This is my first post')
+    end
+  end
+end
```

Comment:
![Screenshot_147](https://github.com/kifle23/blog-app/assets/69638013/f685f3b4-f00c-4567-bf45-1e97381874d0)

- [ ] Please ensure that the tests written for the user and show actions for posts controller are passing